README.md

```diff
@@ -97,3 +98,39 @@ Analyze `production.log` for load+performance statistics about types of requests
 ## [tuning profiles](tuning-profiles)
 
 Some tuning template settings (custom-hiera.yml) for Satellite 6 with [32](tuning-profiles/custom-hiera-medium-32G.yaml), [64](tuning-profiles/custom-hiera-large-64G.yaml), [128](tuning-profiles/custom-hiera-ex-large-128G.yaml) or [256GB](tuning-profiles/custom-hiera-2ex-large-256G.yaml) of RAM. If you have less than 32GB RAM the default settings for Satellite 6 are appropriate.
+
+
+## [check-pulp-msg](check-pulp-msg)
+
+Script that will analyze the qpid and pulp queues in order to help us to identify some issue or weird behavior. Also, it's possible to log the output just changing the variable on the script as below
```

Comment:
```suggestion
This script will check pulp's `resource_manager` and `reserved_resource_worker` qpid queues and print some information to stdout; if the environment variable `CHECK_PULP_MSG_LOG_OUTPUT=Y` is set, the output will also be written to a log at `/var/log/pulp_queue.log`

The output will show the count and total number of bytes for messages currently in the queue, as well as total messages coming into and going out of the queue since the queue came online. The output was also display the number of connections and bindings for each queue.
```

I tried to make this very clear and informative, let me know what you think about it @waldirio 

I suggested the change to `CHECK_PULP_MSG_LOG_OUTPUT` variable name because it's more specific and clearer to the sysadmin if they come across it later. If we go with that change it needs to be updated in the script also.

I wondered if we should make the logfile location configurable with a similar variable, but perhaps that doesn't work if you intend to collect the log file in sos if it is present, and write any insights rules to parse it for discrepancies.

Reply:
We do already collect various `qpid-stat` output in sos, but only at the time the sosreport is generated.

Thinking about it, I would expect that to be good enough for an insights rule to detect most issues. This script would remain an useful way for sysadmins or support to check the data over time without having to generate a complete sosreport, and in that case, maybe it does make sense to have the logfile configurable.